dask/base.py

```diff
@@ -498,18 +481,24 @@ def persist(*args, **kwargs):
 
     try:
         from distributed.client import default_client
-        collections = default_client().persist(collections, **kwargs)
-        if isinstance(collections, list):  # distributed is inconsistent here
-            collections = tuple(collections)
+    except ImportError:
+        pass
+    else:
+        try:
+            client = default_client()
+        except ValueError:
```

Comment:
Yes, having gone through this experience I now agree.  However I've also learned to avoid changing both libraries simultaneously if I can avoid it :)